src/Models/Link.php

```diff
@@ -33,6 +36,15 @@ class Link extends DataObject implements JsonData, Type
         'OpenInNew' => 'Boolean',
     ];
 
+    private static array $has_one = [
+        'LinkArea' => LinkArea::class
+    ];
```

Comment:
```suggestion
    private static array $has_one = [
        'Owner' => DataObject::class
    ];
```
As per above comments. Again, probably this should be `Parent`, but I'll leave everything as `Owner` for this pass.